AWSAuthSDK/Sources/AWSMobileClient/Internal/AWSCognitoAuth+Extensions.h

```diff
@@ -13,9 +13,8 @@
 // permissions and limitations under the License.
 //
 
-#import <AWSMobileClient/AWSCognitoAuth.h>
```

Comment:
What prompted this change? I think I agree with it, but am curious to know what the side effects are.

Reply:
This file will be used by both target AWSMobileClient and AWSMobileClientXCF, so the import doesnot work. Looking at the content below we donot need the whole file imported and just `AWSCognitoAuthConfiguration` works.

I do not see any side effect because this is internally used by AWSMobileClient class where the header file is already imported in `AWSMobileClient.h` and `AWSMobileClientXCF.h` file.